src/ocaml-config/ocaml_config.ml

```diff
@@ -340,7 +340,9 @@ let make vars =
     let os_type                  = get vars "os_type" in
     let standard_library_default = get vars "standard_library_default" in
     let standard_library         = get vars "standard_library" in
-    let standard_runtime         = get vars "standard_runtime" in
+    let standard_runtime =
+      Option.value (get_opt vars "standard_runtime") ~default:"-"
```

Comment:
is "-" a valid value, or is this just a placeholder? (ie: should this be an option instead?)

Reply:
It should probably be an option yeah. It's probably better to fail if the user tries to use this variable.

Reply:
Yes, I also think it would be best to fail. Ideally, it would be best for the error message to be useful as well - tell the user the variable has been removed.

Reply:
Actually, I think that this variable is obscure enough that we can just remove it. Technically, it's a breaking change, but if nobody is using it on opam (should be easy enough to grep) we don't have to bother.

Reply:
I just updated the default value to make things more explicit. Other variables might get removed in the furture, so we should have a general mechanism to deal with it, but for now this should do.